src/api/app/views/webui/image_templates/index.html.haml

```diff
@@ -23,7 +23,9 @@
                 %span.name.break-words
                   = link_to_if(!remote, title_or_name(template), package_show_path(project: project, package: template))
                 %span.description.grey.break-words
-                  = template.description
+                  -# Remote templates with empty description
+                  - unless template.description == '{}'
```

Comment:
Hm, shouldn't you catch this wherever you calculate the string?

Reply:
Fair enough :thumbsup: